internal/service/cognitoidentity/generate.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) HashiCorp, Inc.
 // SPDX-License-Identifier: MPL-2.0
 
-//go:generate go run ../../generate/tags/main.go -ListTags -ServiceTagsMap -UpdateTags
+//go:generate go run ../../generate/tags/main.go -AWSSDKVersion=2 -ListTags -ServiceTagsMap -UpdateTags
```

Comment:
Is there a way to make it so the tags aren't pointers? I manually modified the tags_gen file because I wasn't sure.